cmd/mt-gateway/main.go

```diff
@@ -47,14 +51,26 @@ func main() {
 		fmt.Println("Flags:")
 		flag.PrintDefaults()
 	}
-	flag.Parse()
+
+	// Only try and parse the conf file if it exists
+	path := ""
+	if _, err := os.Stat(*confFile); err == nil {
+		path = *confFile
+	}
+	conf, err := globalconf.NewWithOptions(&globalconf.Options{
+		Filename:  path,
+		EnvPrefix: "MT_",
+	})
+	if err != nil {
+		log.Fatalf("error with configuration file: %s", err)
```

Comment:
the message is saying there's an error with the configuration file, but couldn't it also be an error with the env variable values? 

```suggestion
		log.Fatalf("error with configuration: %s", err)
```

Reply:
Impl looks like file loading is the only thing that can generate an error
![Screen Shot 2020-01-28 at 12 10 22 PM](https://user-images.githubusercontent.com/131809/73287399-34e5e380-41c7-11ea-855a-eadb38f4ea3d.png)